tests/integration/tier0/check-privileges/test_user_privileges.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+
+def test_check_user_privileges(shell):
+    user = "testuser"
+    # Create non-root user if not created already
+    os.system(f"useradd '{user}'")
+    # Set user to non-root entity 'testuser' and run c2r
+    # the program should exit as it is required to be run by root
+    assert shell("runuser -l testuser -c 'convert2rhel'").returncode != 0
```

Comment:
I would like to make sure that we know exactly why convert2rhel ended with non zero return value. Something like this:
```suggestion
    result = shell("runuser -l testuser -c 'convert2rhel'")
    assert result.returncode != 0
    assert result.output = "convert2rhel requires root privileges" 
```
I don't know the exact string, but something that would check that the correct behavior.

Reply:
Good point, I will change that.